deploy/podman/README-disconnected.md

```diff
@@ -0,0 +1,151 @@
+$ podman tag quay.io/edge-infrastructure/assisted-installer:latest <container image registry server:port>/edge-infrastructure/assisted-installer:latest
+$ podman tag quay.io/edge-infrastructure/assisted-installer-controller:latest <container image registry server:port>/edge-infrastructure/assisted-installer-controller:latest
+$ podman push <container image registry server:port>/edge-infrastructure/assisted-installer-agent:latest
+$ podman push <container image registry server:port>/edge-infrastructure/assisted-installer:latest
+$ podman push <container image registry server:port>/edge-infrastructure/assisted-installer-controller:latest
+```
+
+## Identify a Web Server for ISO mirroring
+
+You will need to have a Web server available within your disconnected environment. The Assisted Installer requires a URL to retrieve the RHCOS ISO from on the fly. If you have an existing web server available in your disconnected environment you can use that to host this file. Otherwise, this section will detail some steps for a Fedora/RHEL box.
+
+We need to host the RHCOS image required for booting. We will use NGINX to handle this for us. First get a copy of the RHCOS images and place them in /usr/share/nginx/html... 
+
+```shell
+$ sudo dnf install -y nginx
+$ sudo mkdir -p /usr/share/nginx/html/pub/openshift-v4/dependencies/rhcos/4.10/4.10.16
+$ cd /usr/share/nginx/html/pub/openshift-v4/dependencies/rhcos/4.10/4.10.16
+$ sudo wget https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.10/4.10.16/rhcos-4.10.16-x86_64-live.x86_64.iso
+$ sudo wget https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.10/4.10.16/sha256sum.txt
+```
+
+If you installed a web server on the assisted installer host, you will need to configure the firewall:
+
+```shell
+$ sudo firewall-cmd --permanent --add-port={80/tcp,443/tcp}
+$ sudo firewall-cmd --reload
+$ sudo systemctl enable nginx --now
+```
+
+## Running the Assisted Installer via podman
+
+You will need to configure multiple files to properly use the Assisted Installer in a disconnected environment. You will need to gather up the following information before proceeding:
+
+* signing cert for your container image registry
+* host name for your container image registry
+* hostname or IP address for the web server hosting the RHCOS ISO file (if not using the assisted installer host)
+
+### Install the Assisted Installer Service
+
+Start by creating a directory for the assistedInstaller and copy the `pod-persistent-disconnected.yml` and `configmap-disconnected.yml` files from this Git repo directory into the assistedInstaller directory:
+
+```shell
+$ mkdir ~/assistedInstaller
+$ cp pod-persistent-disconnected.yml configmap-disconnected.yml ~/assistedInstaller
+```
+
+2. Create a registry.conf file and change all "\<container image registry server:port\>" entries to point to your assisted installer host name
```

Comment:
What about patching install config? shouldnt it be added there as well? how would it get to OCP otherwise?

Another question - what if that registry has certificates - worth covering it as well IMO

Reply:
@romfreiman I have made a few updates to the document to address these concerns. I have added an appendix with information on how to retrieve a self-signed signing cert to apply to the configuration, and I have added additional details in the Creating the Cluster section with links to the documentation on how to add the required configurations when using the Web UI (has to be done by API), when using aicli (not sure if you want this in here or not) and the API. Let me know if you have additional questions.